rfc/0084-app-service-level-telemetry.md

```diff
@@ -0,0 +1,559 @@
+      <td>&lt;=1s<br><code style="white-space:nowrap">le="1000"</code></td>
+      <td>&lt;=2.5s<br><code style="white-space:nowrap">le="2500"</code></td>
+      <td>&gt;2.5s<br><code style="white-space:nowrap">le="+Inf"</code></td>
+    </tr>
+    <tr>
+      <td>sdk_kv_mutation_durable</td>
+      <td>&lt;=10ms<br><code style="white-space:nowrap">le="10"</code></td>
+      <td>&lt;=100ms<br><code style="white-space:nowrap">le="100"</code></td>
+      <td>&lt;=500ms<br><code style="white-space:nowrap">le="500"</code></td>
+      <td>&lt;=1s<br><code style="white-space:nowrap">le="1000"</code></td>
+      <td>&lt;=2s<br><code style="white-space:nowrap">le="2000"</code></td>
+      <td>&lt;=10s<br><code style="white-space:nowrap">le="10000"</code></td>
+      <td>&gt;10s<br><code style="white-space:nowrap">le="+Inf"</code></td>
+    </tr>
+    <tr>
+      <td>sdk_query</td>
+      <td></td>
+      <td>&lt;=100ms<br><code style="white-space:nowrap">le="100"</code></td>
+      <td>&lt;=1s<br><code style="white-space:nowrap">le="1000"</code></td>
+      <td>&lt;=10s<br><code style="white-space:nowrap">le="10000"</code></td>
+      <td>&lt;=30s<br><code style="white-space:nowrap">le="30000"</code></td>
+      <td>&lt;=75s<br><code style="white-space:nowrap">le="75000"</code></td>
+      <td>&gt;75s<br><code style="white-space:nowrap">le="+Inf"</code></td>
+    </tr>
+    <tr>
+      <td>sdk_search</td>
+      <td></td>
+      <td>&lt;=100ms<br><code style="white-space:nowrap">le="100"</code></td>
+      <td>&lt;=1s<br><code style="white-space:nowrap">le="1000"</code></td>
+      <td>&lt;=10s<br><code style="white-space:nowrap">le="10000"</code></td>
+      <td>&lt;=30s<br><code style="white-space:nowrap">le="30000"</code></td>
+      <td>&lt;=75s<br><code style="white-space:nowrap">le="75000"</code></td>
+      <td>&gt;75s<br><code style="white-space:nowrap">le="+Inf"</code></td>
+    </tr>
+    <tr>
+      <td>sdk_analytics</td>
+      <td></td>
+      <td>&lt;=100ms<br><code style="white-space:nowrap">le="100"</code></td>
+      <td>&lt;=1s<br><code style="white-space:nowrap">le="1000"</code></td>
+      <td>&lt;=10s<br><code style="white-space:nowrap">le="10000"</code></td>
+      <td>&lt;=30s<br><code style="white-space:nowrap">le="30000"</code></td>
+      <td>&lt;=75s<br><code style="white-space:nowrap">le="75000"</code></td>
+      <td>&gt;75s<br><code style="white-space:nowrap">le="+Inf"</code></td>
+    </tr>
+    <tr>
+      <td>sdk_eventing</td>
+      <td></td>
+      <td>&lt;=100ms<br><code style="white-space:nowrap">le="100"</code></td>
+      <td>&lt;=1s<br><code style="white-space:nowrap">le="1000"</code></td>
+      <td>&lt;=10s<br><code style="white-space:nowrap">le="10000"</code></td>
+      <td>&lt;=30s<br><code style="white-space:nowrap">le="30000"</code></td>
+      <td>&lt;=75s<br><code style="white-space:nowrap">le="75000"</code></td>
+      <td>&gt;75s<br><code style="white-space:nowrap">le="+Inf"</code></td>
+    </tr>
```

Comment:
Is there supposed to be a histogram for `sdk_management` too?

Reply:
Thank you. I have updated the table